### ISSUE TITLE
impossible to deactivate encryption 

### ISSUE THREAD

**Issue by webratte** [k9mail/k-9#4378]
Please search to check for an existing issue (including closed issues, for which the fix may not have yet been released) before opening a new issue: https://github.com/k9mail/k-9/issues?q=is%3Aissue

### Expected behavior
If I replay to a encrypted but not signed mail it should be possible to deaktivate encryption.

### Actual behavior
Click on the red lock do nothing.

### Steps to reproduce
1. receive a encrypted but NOT signed mail.
2. click the reply button.
3. Click the red lock symbol.
4. Encryption can't be deactivated.

### Environment
K-9 Mail version: 5.702

Android version: 7.0

Account type (IMAP, POP3, WebDAV/Exchange): IMAP

Please take some time to [retrieve logs](https://github.com/k9mail/k-9/wiki/LoggingErrors) and attach them here:

**Comment by webratte** [comment]
I use the feature of mailbox.org to encrypt all incoming mails.
This mails are not signed.

To reply to this mails I have to disable encryption.

So if I can't disable encryption for replying a mail k9 will be absolutely not usable for me.

**Comment by yoshimo** [comment]
I assume you Mailbox.org uses a public key you gave them before hand.
If you have the private key for that public key on your mobile phone you should be able to read the mail, which you seem to, and only use encryption if the receiver of the reply has a public key which you know. So where exactly is the issue?

**Comment by webratte** [comment]
Let me explain:
-I got a unencrypted mail
-mailbox.org encrypt that mail with my public key
-With k9 I can decrypt and read this mail.

To this point all it's ok.
But now I like to reply.

If I now click reply k9 try to encrypt but say there is no valid encryption key for the receiver. And I can't send this mail.

I  the current stable k9 version from play store I can deactivate decryption for this reply.

In the version i mentioned in my first post it's impossible.

Do you understand?

**Comment by TheBiostarfly** [comment]
I'm experiencing the same issue. Was hoping it got resolved with the last update, but I still can't reply to encrypted emails nor deactivate the encryption.

**Comment by stuart12** [comment]
I too cannot disable encryption with K-9 Mail 5.707 on LineageOS 16.0.  On my main account, all my incoming email is encrypted but not signed by my MDA. I have OpenKeychain setup so that I can read my emails. I never send encrypted emails. After I press reply, there is a red padlock containing a cross on the right of the From address. When I click on the arrow to send the email I get a toast saying **No key configured for this account! Check your settings**. I then tap the red padlock to disable encryption and I get the message:

```
Encryption not possible

Some of the selected recipients don't support this feature!

Back      Disable Encryption
```
I then click on **Disable Encryption** but the red padlock remains and I get the same toast if I then click on send.  I can never send my reply.

I can however send new emails and reply to non-encrypted emails (in a different account on a different server).

On K-9 Mail version 5.600 clicking on the red padlock gives the same **Encryption not possible** message but with this version, clicking on **Disable Encryption** removes the red padlock and I can then send my reply.

I note that on [F-Droid](https://f-droid.org/packages/com.fsck.k9/), Version 5.600 added on 2018-09-11 is the suggested version.  Is this the version we should be using?

**Comment by muety** [comment]
I got the exact same problem. Any news on this?

**Comment by TheBiostarfly** [comment]
> I got the exact same problem. Any news on this?

The status hasn't changed. I'm wondering if there's a way to push this request.

**Comment by cketti** [comment]
Apologies. Once pull request #4772 is merged the "disable encryption" button should actually do so.

**Comment by webratte** [comment]
Great to read it's fixed now.

I'm looking forward to the big release with the new UI 👍

Thanks you very much.
